packages/editor/src/components/DataSource/ApiForm/Response.tsx

```diff
@@ -57,19 +63,17 @@ export const Response: React.FC<Props> = props => {
         </h2>
         <AccordionPanel pb={4} padding={0} height="250px">
           <Flex alignItems="center" justifyContent="center" height="100%">
-            {props.loading ? (
+            {props.loading || !isOpen ? (
               <Spinner />
             ) : (
               <CodeEditor
                 className={css`
                   width: 100%;
                   height: 100%;
                 `}
-                mode={{
-                  name: 'javascript',
-                  json: true,
-                }}
+                mode="text/plain"
```

Comment:
maybe JSON is the expected display mode?

Reply:
ok, I changed back.